test/api/handlers/publishedProjects/get-one.js

```diff
@@ -0,0 +1,74 @@
+var Lab = require('lab');
+var lab = exports.lab = Lab.script();
+
+var experiment = lab.experiment;
+var test = lab.test;
+var before = lab.before;
+var after = lab.after;
+var expect = require('code').expect;
+
+var config = require('../../../lib/fixtures/publishedProjects').getOne;
+var server;
+
+before(function(done) {
+  require('../../../lib/mocks/server')(function(testServer) {
+    server = testServer;
+
+    config(function(err, getOne) {
+      if (err) { throw err; }
+
+      config = getOne;
+      done();
+    });
+  });
+});
+
+after(function(done) {
+  server.stop(done);
+});
+
+// GET /publishedProjects/:publishedProject_id
+experiment('[Get one published project]', function() {
+  test('success case', function(done) {
+    var options = config.success.default;
+
+    server.inject(options, function(response) {
+      expect(response.statusCode).to.equal(200);
+
+      expect(response.result).to.exist();
+      expect(response.result.id).to.be.a.number();
+      expect(response.result.title).to.be.a.string();
+      expect(response.result.tags).to.be.a.string();
+      expect(response.result.description).to.be.a.string();
+      expect(response.result.date_created).to.be.a.string();
```

Comment:
having dates be strings is still very worrisome. We may want to fix this sooner than later =(